packages/ui-kit/src/components/Counter/Counter.tsx

```diff
@@ -73,7 +75,7 @@ export const CounterComponent = React.forwardRef<HTMLSpanElement, CounterProps>(
           return
         }
 
-        if (!isStatic && (error?.name === 'AccessTokenError' || !query?.metric || !query?.timeRange)) {
+        if (!isStatic && (error?.name === 'AccessTokenError' || !query?.metric)) {
           // console.error(
           //   'InvalidPropsError: When opting for fetching data you must pass at least `accessToken`, `metric` and `timeRange` in the `query` prop'
           // ) we will set logs as a feature later
```

Comment:
Out of date. Delete? Or actually log using configured log level?

Reply:
I think we can use our logger here